README.md

```diff
@@ -32,159 +30,89 @@ A version of the Swagger Petstore API is displayed by default.
-- particular release, for example `v1.2.0`: https://rebilly.github.io/ReDoc/releases/v1.2.0/redoc.min.js
-- `v1.x.x` release: https://rebilly.github.io/ReDoc/releases/v1.x.x/redoc.min.js
-- `latest` release: https://rebilly.github.io/ReDoc/releases/latest/redoc.min.js - points to latest 1.x.x release since 2.x releases are not hosted on this CDN but on unpkg.
+## Usage
 
-## Version Guidance
-| Redoc Release | OpenAPI Specification |
-|:--------------|:----------------------|
-| 2.0.0-alpha.54| 3.1, 3.0.x, 2.0       |
-| 2.0.0-alpha.x | 3.0, 2.0              |
-| 1.19.x        | 2.0                   |
-| 1.18.x        | 2.0                   |
-| 1.17.x        | 2.0                   |
+Redoc is provided as a CLI tool (also distributed as a Docker image), HTML tag, and React component.
 
-## Showcase
-- [Rebilly](https://api-reference.rebilly.com/)
-- [Docker Engine](https://docs.docker.com/engine/api/v1.25/)
-- [Zuora](https://www.zuora.com/developer/api-reference/)
-- [Discourse](http://docs.discourse.org)
-- [Commbox](https://www.commbox.io/api/)
-- [APIs.guru](https://apis.guru/api-doc/)
-- [BoxKnight](https://www.docs.boxknight.com/)
+### Generate documentation from the CLI
 
-## Lint OpenAPI definitions
+If you have Node installed, quickly generate documentation using `npx`:
 
-Redocly CLI is an [open source command-line tool](https://github.com/Redocly/redocly-cli) that you can use to lint
-your OpenAPI definition. Linting helps you to catch errors and inconsistencies in your
-OpenAPI definition before publishing.
+```
+npx @redocly/cli build-docs openapi.yaml 
+```
 
-Learn more about [API standards and linting](https://redocly.com/docs/cli/api-standards/) in the main Redocly documentation.
+The tool outputs by default to a file named `redoc-static.html` that you can open in your browser.
 
-## Deployment
+> [Redocly CLI](https://github.com/Redocly/redocly-cli/) does more than docs, check it out and add linting, bundling and more to your API workflow.
 
-### TL;DR final code example
+### Add an HTML element to the page
 
-To render your OpenAPI definition using Redoc, use the following HTML code sample and
-replace the `spec-url` attribute with the url or local file address to your definition.
+Create an HTML page, or edit an existing one, and add the following:
 
 ```html
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Redoc</title>
-    <!-- needed for adaptive design -->
-    <meta charset="utf-8"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700|Roboto:300,400,700" rel="stylesheet">
-
-    <!--
-    Redoc doesn't change outer page styles
-    -->
-    <style>
-      body {
-        margin: 0;
-        padding: 0;
-      }
-    </style>
-  </head>
-  <body>
-    <redoc spec-url='http://petstore.swagger.io/v2/swagger.json'></redoc>
+    <redoc spec-url="http://petstore.swagger.io/v2/swagger.json"></redoc>
```

Comment:
I would keep the whole html here. It may be simpler for some people to understand.

Reply:
Would it work to put the full example on the deployment page? This snippet should be enough for quick reference, but the whole example does need to exist somewhere easy to find. Although currently the HTML install instructions are like this https://redocly.com/docs/redoc/deployment/html/ which seems like it refers to something different. I think I have more to do here!

Reply:
It would work.